localstack/testing/pytest/validation_tracking.py

```diff
@@ -42,7 +43,9 @@ def find_validation_data_for_item(item: pytest.Item) -> Optional[dict]:
 
 def record_passed_validation(item: pytest.Item, timestamp: Optional[datetime.datetime] = None):
     base_path = os.path.join(item.fspath.dirname, item.fspath.purebasename)
-    with open(f"{base_path}.validation.json", "w+") as fd:
+    file_path = f"{base_path}.validation.json"
+    Path(file_path).touch()
+    with open(file_path, "r+") as fd:
```

Comment:
And I missed that again when I encountered the same issue with S3 😅 
I did it in a try/except there because that's what `touch` does under the hood, but this is shorter! 
```suggestion
    file_path = Path(f"{base_path}.validation.json")
    file_path.touch()
    with file_path.open(mode="r+") as fd:
```

Reply:
yeah I noticed it when creating the -ext files :grin: